### PR TITLE
DOC/FAQ: update RDMA requirements

### PR DESCRIPTION
**Pull request by evgeny-leksikov**
## What
update RDMA requirements in FAQ

## Why ?
sync with https://github.com/openucx/ucx/pull/8403

### CODE REVIEW

**Review by yosefe — APPROVED** [review]

**Comment by shamisp** [comment]
Can you please clarify starting from what version the requirement is relevant ? There are bunch of ppl that just don't have the luxury for easy upgrade for OS kernel, etc.

**Comment by evgeny-leksikov** [comment]
> Can you please clarify starting from what version the requirement is relevant ? There are bunch of ppl that just don't have the luxury for easy upgrade for OS kernel, etc.

updated

**Comment by shamisp** [comment]
@evgeny-leksikov @edgargabriel can you please add similar statement for ROCM and CUDA sw stack versioning ?

**Comment by evgeny-leksikov** [comment]
> @evgeny-leksikov @edgargabriel can you please add similar statement for ROCM and CUDA sw stack versioning ?

@shamisp @Akshay-Venkatesh CUDA requirements are updated. @edgargabriel could you please privide ROCM requirements?

**Comment by edgargabriel** [comment]
@evgeny-leksikov I double checked with our runtime team, we agreed on listing a required rocm version >=4.0 as a prerequisite. Can you please add it to the PR, it might be easier than me filing a PR against your PR.

**Comment by evgeny-leksikov** [comment]
> @evgeny-leksikov I double checked with our runtime team, we agreed on listing a required rocm version >=4.0 as a prerequisite. Can you please add it to the PR, it might be easier than me filing a PR against your PR.

@edgargabriel thanks for update, pls review exact wording and let me know if it should be changed by some reason

**Review by edgargabriel — APPROVED** [review]

**Comment by evgeny-leksikov** [comment]
@Akshay-Venkatesh could you please confirm correctness of CUDA requirements?

**Review by Akshay-Venkatesh — DISMISSED** [review]

**Comment by evgeny-leksikov** [comment]
@shamisp @tonycurtis @yosefe pls take a look

File: docs/source/faq.md [hunk 1]
```diff
@@ -78,11 +78,11 @@ for various features accordingly.
 If some of the modules UCX was built with are not found during runtime, they will
 be silently disabled.
 
-* **Basic shared memory and TCP support** - always enabled
+* **Basic shared memory and TCP support** - always enabled.
 * **Optimized shared memory** - requires knem or xpmem drivers. On modern kernels also CMA (cross-memory-attach) mechanism will be used.
```
Comment (tonycurtis) [inline docs/source/faq.md:82]: On modern kernels, CMA will also be used if available

**Review by tonycurtis — CHANGES_REQUESTED** [review]

File: docs/source/faq.md (hunk 1 above)
Comment (evgeny-leksikov) [reply docs/source/faq.md]: fixed

**Review by yosefe — APPROVED** [review]

**Review by shamisp — APPROVED** [review]

**Pull request merged**